### PR TITLE
Improvements in plugin myfiles

### PR DESCRIPTION
**Pull request by ThomasDumont**

### CODE REVIEW

File: src/java/fr/paris/lutece/plugins/myfiles/web/servlet/Download.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        processRequest( request, response );
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getServletInfo( )
+    {
+        return "Servlet serving stylesheets";
```
Comment (jonenst) [inline src/java/fr/paris/lutece/plugins/myfiles/web/servlet/Download.java]: This is copy pasted from somewhere else :)

**Comment by pierrelevy** [comment]
The Bucket object doesn't seem to me very useful.

File: src/java/fr/paris/lutece/plugins/myfiles/service/storage/impl/MinioFileStorage.java [hunk 2]
```diff
@@ -112,7 +118,9 @@ public MinioFileStorage( )
                     myFile.setName( item.objectName( ) );
                     ObjectStat stat = _client.statObject( strBucketName, item.objectName( ) );
                     myFile.setContentType( stat.contentType( ) );
-                    String strUrl = _client.presignedGetObject( strBucketName, item.objectName( ), 60 * 60 * 24 );
+                    myFile.setSize( stat.length(  ) );
```
Comment (jonenst) [inline src/java/fr/paris/lutece/plugins/myfiles/service/storage/impl/MinioFileStorage.java:121]: Is this really part of this commit ? Or should it be split in another commit

File: webapp/WEB-INF/conf/plugins/myfiles_context.xml [hunk 3]
```diff
@@ -5,6 +5,8 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd" >
 
 
-       <bean id="myfiles.storage" class="fr.paris.lutece.plugins.myfiles.service.storage.impl.MinioFileStorage" />
+    <bean id="myfiles.storage" class="fr.paris.lutece.plugins.myfiles.service.storage.impl.DatabaseFileStorage" />
```
Comment (jonenst) [inline webapp/WEB-INF/conf/plugins/myfiles_context.xml]: Do we really want to switch to this by default ?

Comment (ThomasDumont) [reply webapp/WEB-INF/conf/plugins/myfiles_context.xml]: As Minio server need to be configured before installing the plugin, it seems better to switch to a local implementation by default. In order to highlight this point, I put this change in its own commit.

File: src/java/fr/paris/lutece/plugins/myfiles/service/storage/impl/MinioFileStorage.java (hunk 2 above)
Comment (ThomasDumont) [reply src/java/fr/paris/lutece/plugins/myfiles/service/storage/impl/MinioFileStorage.java:121]: Yes, it should be as it's not related to the commit message. Thus, size isn't currently displayed in home view. I made a new commit to add this.

**Comment by ThomasDumont** [comment]
Yes Bucket object is currently not very useful as it just has one field.
So I've made a new commit to remove the Bucket object.
